src/createTokenBridge.ts

```diff
@@ -171,6 +209,14 @@ export async function createTokenBridge<
 }: CreateTokenBridgeParams<TParentChain, TOrbitChain>): Promise<
   CreateTokenBridgeResults<TParentChain, TOrbitChain>
 > {
+  await assertTokenBridgeDoesntExist({
+    parentChainPublicClient,
+    orbitChainPublicClient,
+    tokenBridgeCreatorAddress:
+      tokenBridgeCreatorAddressOverride ?? getTokenBridgeCreatorAddress(parentChainPublicClient),
```

Comment:
i see that `?? getTokenBridgeCreatorAddress(parentChainPublicClient)` is already in `assertTokenBridgeDoesntExist`, seems redundant to have it here